components/develop/modules/ROOT/pages/sfuel/gas-vs-sfuel.adoc

```diff
@@ -0,0 +1,37 @@
+= Gas vs sFUEL
+
+== What is gas? 
+
+Gas is an asset or a resource used to run something. It can be the fuel needed to enable a car to run or a tokenized asset that enables a blockchain network to run.
+
+From link:https://ethereum.org/en/developers/docs/gas/#what-is-gas[https://ethereum.org] --
+
+"Gas refers to the unit that measures the amount of computational effort required to execute specific operations on the Ethereum network."
+
+
+[width="100%"]
+== Gas vs sFUEL
+|======
+|                | +++<strong>Gas</strong>+++			| +++<strong>sFUEL</strong>+++
```

Comment:
suggest Gas (Ethereum) and sFUEL (SKALE) headers
also consider using header rows instead of explicit <strong> see: https://docs.asciidoctor.org/asciidoc/latest/tables/add-header-row/#explicitly-assign-header-to-the-first-row

Reply:
Resolved by b9b742bab300abfb2a70e49e06d0e82912a6c5da